ext/ldap/config.m4

```diff
@@ -180,6 +194,71 @@ if test "$PHP_LDAP" != "no"; then
      AC_DEFINE(HAVE_ORALDAP,1,[ ])
      AC_DEFINE(HAVE_ORALDAP_10,1,[ ])
 
+     AC_DEFINE(HAVE_ORALDAP_10,1,[ ])
+
+  elif test -f $LDAP_LIBDIR/$MACHINE_INCLUDES/liblber.a || test -f $LDAP_LIBDIR/$MACHINE_INCLUDES/liblber.$SHLIB_SUFFIX_NAME; then
```

Comment:
Is there any better way of doing this than copy&pasting the whole code block with `$MACHINE_INCLUDES` added? Is it possible to cover machine-includes/no-machine-includes in a loop or something?